### ISSUE TITLE
cannot resolve on project with space on it's path

### ISSUE THREAD

**Issue by davinpra** [googlesamples/unity-jar-resolver#108]
as the title said, if your project have a space on it's path then resolver would fail

**Comment by stewartmiles** [comment]
@davinpra could you share a log of the failure you're seeing?

**Comment by davinpra** [comment]
project folder with space
```
`Gradle failed to fetch dependencies.

Failed to run 'D:\Unity Project\Test Resolver\Temp\PlayServicesResolverGradle\gradlew.bat -b "D:\Unity Project\Test Resolver\Temp\PlayServicesResolverGradle\PlayServicesResolver.scripts.download_artifacts.gradle" --no-daemon "-PANDROID_HOME=D:\adt-bundle-windows-x86-20130219-complete\adt-bundle-windows-x86-20130219-complete\sdk\" "-PTARGET_DIR=D:\Unity Project\Test Resolver\Assets\Plugins\Android" "-PMAVEN_REPOS=file:///D:/Unity%20Project/Test%20Resolver/Assets/GooglePlayGames/Editor/m2repository" "-PPACKAGES_TO_COPY=com.google.games:gpgs-plugin-support:0.9.50"'
stdout:
Download https://maven.google.com/com/android/support/support-v4/25.2.0/support-v4-25.2.0.pom
Download https://maven.google.com/com/android/support/support-compat/25.2.0/support-compat-25.2.0.pom
Download https://maven.google.com/com/android/support/support-core-ui/25.2.0/support-core-ui-25.2.0.pom
Download https://maven.google.com/com/android/support/support-fragment/25.2.0/support-fragment-25.2.0.pom
Download https://maven.google.com/com/android/support/support-core-utils/25.2.0/support-core-utils-25.2.0.pom
Download https://maven.google.com/com/android/support/support-media-compat/25.2.0/support-media-compat-25.2.0.pom
Download https://maven.google.com/com/android/support/support-annotations/25.2.0/support-annotations-25.2.0.pom
Download https://maven.google.com/com/android/support/support-v4/25.2.0/support-v4-25.2.0.aar
Download https://maven.google.com/com/android/support/support-fragment/25.2.0/support-fragment-25.2.0.aar
Download https://maven.google.com/com/android/support/support-media-compat/25.2.0/support-media-compat-25.2.0.aar
Download https://maven.google.com/com/android/support/support-core-utils/25.2.0/support-core-utils-25.2.0.aar
Download https://maven.google.com/com/android/support/support-core-ui/25.2.0/support-core-ui-25.2.0.aar
Download https://maven.google.com/com/android/support/support-compat/25.2.0/support-compat-25.2.0.aar
Download https://maven.google.com/com/android/support/support-annotations/25.2.0/support-annotations-25.2.0.jar

stderr:

FAILURE: Build failed with an exception.

* What went wrong:
Task 'Project\Test' not found in root project 'PlayServicesResolverGradle'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1m 5s

exit code: 1`
```

project folder with space on it's path

```
Gradle failed to fetch dependencies.

Failed to run 'D:\Unity Project\TestResolver\Temp\PlayServicesResolverGradle\gradlew.bat -b "D:\Unity Project\TestResolver\Temp\PlayServicesResolverGradle\PlayServicesResolver.scripts.download_artifacts.gradle" --no-daemon "-PANDROID_HOME=D:\adt-bundle-windows-x86-20130219-complete\adt-bundle-windows-x86-20130219-complete\sdk\" "-PTARGET_DIR=D:\Unity Project\TestResolver\Assets\Plugins\Android" "-PMAVEN_REPOS=file:///D:/Unity%20Project/TestResolver/Assets/GooglePlayGames/Editor/m2repository" "-PPACKAGES_TO_COPY=com.google.games:gpgs-plugin-support:0.9.50"'
stdout:

stderr:

FAILURE: Build failed with an exception.

* What went wrong:
Project 'Project\TestResolver\Assets\Plugins\Android -PMAVEN_REPOS=file' not found in root project 'PlayServicesResolverGradle'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s

exit code: 1
```

**Comment by GrenadeJane** [comment]
Hi, I've a relative issue here, I've got a '#' in my project's path, and the resolve just don't succeeded. After remove this specific character, everything is working. 
( OSX High Sierra, Firebase Unity SDK 4.5.0, Unity 2017.2.0f3 )

**Comment by stewartmiles** [comment]
This should be fixed in 4fc7083877b642d045e06204824475b57f5e4d92 please give it a try.
